packages/process/process-notify.pony

```diff
@@ -0,0 +1,30 @@
+interface ProcessNotify
+  """
+  Notifications for Process connections.
+  """
+    
+  fun ref stdout(data: Array[U8] iso) =>
+    """
+    ProcessMonitor calls this when new data is received on STDOUT of the 
+    forked process
+    """
+    
+  fun ref stderr(data: Array[U8] iso) =>
+    """
+    ProcessMonitor calls this when new data is received on STDERR of the 
+    forked process
+    """
+
+  fun ref failed(err: ProcessError) =>
+    """
+    ProcessMonitor calls this if we run into errors communicating with the
+    forked process.
+    """
+    None
```

Comment:
why do 2 of these `failed` and `dispose` have None but `stderr` and `stdout` don't.

intentional or oversight?